test/runtests.jl

```diff
@@ -1,6 +1,8 @@
 using Base.Broadcast: broadcastable
 using ChainRules
 using ChainRulesCore
+using ChainRulesTestUtils
+using ChainRulesTestUtils: _fdm
```

Comment:
we should open an issue on ChainRulesTestUtils about whether or not we want this to be exported / part of the public API